refactoring to use external `axisregistry` module

felipesanches opened the pull request:
https://github.com/googlefonts/axisregistry
(issue #515)

felipesanches commented:
this one is also ready for review

m4rc1e commented:
I see folks are still treating the google/fonts repo as the source for the axis_reg since they're opening proposals there. I think I'll merge this once gf eng is aware of the repo. I guess we could bring this up the upcoming meeting with them?

m4rc1e commented:
Once this is done I'll cut 0.9.1

felipesanches commented:
> Just fix [googlefonts/axisregistry#7](https://github.com/googlefonts/axisregistry/issues/7) and we're good.

Fixed it now, released `axisregistry==0.1.1` and updated this PR to use it. Ready for yet another review. Thanks!

felipesanches commented:
@m4rc1e actually, please wait a bit more because I am making one additional improvement.

felipesanches commented:
@m4rc1e, OK. Now it is good for a final review. `axisregistry==0.2.0` simplified its API so that now the diff in this PR is minimal ;-)

m4rc1e reviewed (approved):
LGTM. Thanks.
